WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -19,6 +19,8 @@ extension UserDefaults {
         case analyticsUsername
         case notificationsLastSeenTime
         case notificationsMarkAsReadCount
+        case statsVersionLastSeen
```

Comment:
Do these two values depend on the account the user is logged in? If that's the case, I think we should manage them (save and retrieve them) via the AppSettingsStore, similarly to the way we save and retrieve a user's last selected shipment tracking provider. As en extra benefit, those values would get cleared when login out.

Reply:
> Do these two values depend on the account the user is logged in?

ohh cool I didn't know of `AppSettingsStore` (I thought the app settings are the settings from server side without checking the implementaiton), will update!

> As en extra benefit, those values would get cleared when login out.

this is perfect! I felt a bit uncomfortable when I had to manually reset the stats user defaults in `DefaultStoresManager`